src/icalendar/attr.py

```diff
@@ -1,24 +1,316 @@
+        "STANDARD" and "DAYLIGHT" sub-components of the "VTIMEZONE"
+        calendar component, but it SHOULD NOT be specified more than once.
+        The recurrence set generated with multiple "RRULE" properties is
+        undefined.
+
+    Description:
+        The recurrence rule, if specified, is used in computing
+        the recurrence set.  The recurrence set is the complete set of
+        recurrence instances for a calendar component.  The recurrence set
+        is generated by considering the initial "DTSTART" property along
+        with the "RRULE", "RDATE", and "EXDATE" properties contained
+        within the recurring component.  The "DTSTART" property defines
+        the first instance in the recurrence set.  The "DTSTART" property
+        value SHOULD be synchronized with the recurrence rule, if
+        specified.  The recurrence set generated with a "DTSTART" property
+        value not synchronized with the recurrence rule is undefined.  The
+        final recurrence set is generated by gathering all of the start
+        DATE-TIME values generated by any of the specified "RRULE" and
+        "RDATE" properties, and then excluding any start DATE-TIME values
+        specified by "EXDATE" properties.  This implies that start DATE-
+        TIME values specified by "EXDATE" properties take precedence over
+        those specified by inclusion properties (i.e., "RDATE" and
+        "RRULE").  Where duplicate instances are generated by the "RRULE"
+        and "RDATE" properties, only one recurrence is considered.
+        Duplicate instances are ignored.
+
+        The "DTSTART" property specified within the iCalendar object
+        defines the first instance of the recurrence.  In most cases, a
+        "DTSTART" property of DATE-TIME value type used with a recurrence
+        rule, should be specified as a date with local time and time zone
+        reference to make sure all the recurrence instances start at the
+        same local time regardless of time zone changes.
+
+        If the duration of the recurring component is specified with the
+        "DTEND" or "DUE" property, then the same exact duration will apply
+        to all the members of the generated recurrence set.  Else, if the
+        duration of the recurring component is specified with the
+        "DURATION" property, then the same nominal duration will apply to
+        all the members of the generated recurrence set and the exact
+        duration of each recurrence instance will depend on its specific
+        start time.  For example, recurrence instances of a nominal
+        duration of one day will have an exact duration of more or less
+        than 24 hours on a day where a time zone shift occurs.  The
+        duration of a specific recurrence may be modified in an exception
+        component or simply by using an "RDATE" property of PERIOD value
+        type.
+
+    Examples:
+
+        Daily for 10 occurrences:
```

Comment:
This did not render properly in https://icalendar--814.org.readthedocs.build/en/814/api.html#icalendar.cal.Event.rrules, so let's try this. White space is super nitpicky in reStructuredText.

```suggestion
    Examples:
        Daily for 10 occurrences:
```